workspaces/ui-v2/src/pages/docs/EndpointRootPage.tsx

```diff
@@ -471,28 +472,40 @@ export const EndpointRootPage: FC<
                             {(selectedFieldId, setSelectedFieldId) => (
                               <div className={classes.bodyDetails}>
                                 <div>
-                                  <ContributionsList
-                                    renderField={(field) => (
-                                      <DocFieldContribution
-                                        key={
-                                          field.contribution.id +
-                                          field.contribution.contributionKey
-                                        }
-                                        endpoint={{
-                                          pathId,
-                                          method,
-                                        }}
-                                        name={field.name}
-                                        shapes={field.shapes}
-                                        depth={field.depth}
-                                        id={field.fieldId}
-                                        initialValue={field.contribution.value}
-                                        required={field.required}
-                                        setSelectedField={setSelectedFieldId}
-                                      />
-                                    )}
-                                    fieldDetails={fields}
-                                  />
+                                  {process.env
+                                    .REACT_APP_FF_FIELD_LEVEL_EDITS !==
+                                  'true' ? (
+                                    <ContributionsList
```

Comment:
It's probably worth setting this for the `QueryParametersPanel` and the request `HttpBodyPanel` in this component as well